about.html

```diff
@@ -163,6 +163,7 @@ <h3 class="subtitle wow fadeInUp" data-wow-delay=".3s" data-wow-duration="500ms"
 
                             <h3 class="subtitle wow fadeInUp" data-wow-delay=".3s" data-wow-duration="500ms">Our Achievements</h3>
                             <ul>
+                                <li><p class="wow fadeInUp" data-wow-delay=".5s" data-wow-duration="500ms">2020 <a href="https://cyberinstitute.kennesaw.edu/seccdc/">SECCDC</a> Third Place; Team Members: Max Harley, Matthew Rinaldi, Cameron Ogle, Adam Van Scyoc, Madeleine Wolfe, and David Houston</p></li>
```

Comment:
So in previous years we wrote it as "SECCDC Regional Competition". I'd either change those to just "SECCDC" as well or change this to the long one. I like shortening things though. And their links should also be updated.

Reply:
adjusted in 7cb7df2